selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/locator/chat/ChatLocator.java

```diff
@@ -52,7 +52,7 @@
   public static final SelenideElement ELEMENT_CHAT_BUTTON_SAVE_ADD_ROOM   =
                                                                         $(byXpath("//*[@id=\"team-modal-form\"]/div[2]/div[2]/a[1]"));
 
-  public static final SelenideElement ELEMENT_CHAT_RESULT_SEARCH_USER     = $(byXpath("/html/body/div[6]"));
+  public static final SelenideElement ELEMENT_CHAT_RESULT_SEARCH_USER     = $(byXpath("/html/body/div[7]"));
```

Comment:
This xpath is too fragile. Please use a better xpath, based on id or css classes.